package.json

```diff
@@ -17,7 +17,7 @@
   },
   "scripts": {
     "dev": "webpack -d --progress --profile --colors --bail --",
-    "serve": "webpack-dev-server -d --inline --https --host 0.0.0.0 --content-base troposphere/assets",
+    "serve": "webpack-dev-server -d --inline --https --content-base troposphere/assets",
```

Comment:
This shouldn't have been removed. It was related to how i had set up my `alias` for calling `npm run serve` (including --host ...) but after testing, it looks like that isn't required.